README.md

```diff
@@ -88,7 +88,7 @@ After you're finished please remove all the comments and instructions!
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 🛠 Live Demo <a name="live-demo"></a>
-[Recipe App on Rails](https://recipeapp-hs5c.onrender.com)
+[Comming soon]()
```

Comment:
- [ ] [OPTIONAL] Instead of fixing the live demo you just removed the link :broken_heart:  Please remove this Live Demo section since you don't have a live link in your readme file +1 to make your readme file descriptive and professional :+1: